### PR TITLE
[Feature][unittest] Recover UT in AlertPluginManagerTest.java [closes: #6619]

### LINKED ISSUE #6619: [Feature][unittest] Recover UT in AlertPluginManagerTest.java
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

In we merge PR https://github.com/apache/dolphinscheduler/pull/6182 by accident, in https://github.com/apache/dolphinscheduler/pull/6182#issuecomment-918220975 we know that we should make this unittest work instead of remove it, this issue aim to make unittest work again in `dolphinscheduler-alert/src/test/java/org/apache/dolphinscheduler/alert/plugin/AlertPluginManagerTest.java`

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Linkaiup**
## Purpose of the pull request

This pull request recover UT in AlertPluginManagerTest.java[closes: #6619].

## Brief change log

*recover  DolphinPluginLoader loadPlugins() Test in AlertPluginManagerTest.java[closes: #6619]*

## Verify this pull request

This change added tests and can be verified as follows:

  - *Manually verified the change by testing locally.*

### CODE REVIEW

**Comment by zhongjiajie** [comment]
Hi @Linkaiup, thank for your contribution.  I think we should better target specific issue when we submit a PR, you could edit you and add text `closes: #6619` to do it.

And @lenboo @CalvinKirs @ruanwenjun @zhuangchong could you please approve to run workflow for this PR?

File: dolphinscheduler-alert/src/test/java/org/apache/dolphinscheduler/alert/plugin/AlertPluginManagerTest.java [hunk 1]
```diff
@@ -53,6 +54,14 @@ public void testLoadPlugins() {
...
         }
 
+
+        DolphinPluginLoader alertPluginLoader = new DolphinPluginLoader(alertPluginManagerConfig, ImmutableList.of(alertPluginManager));
+        try {
+            alertPluginLoader.loadPlugins();
+        } catch (Exception e) {
+            throw new RuntimeException("load Alert Plugin Failed !", e);
+        }
+
```
Comment (CalvinKirs) [inline dolphinscheduler-alert/src/test/java/org/apache/dolphinscheduler/alert/plugin/AlertPluginManagerTest.java]: It is best to do this, `public void testMethod() throws MyException {` 
No need to use try-catch to catch exceptions.

Comment (Linkaiup) [reply dolphinscheduler-alert/src/test/java/org/apache/dolphinscheduler/alert/plugin/AlertPluginManagerTest.java]: Thanks for your advice ! @CalvinKirs https://github.com/apache/dolphinscheduler/pull/6674/commits/341d9aa50652da02ef85766390388f9a2c3c1ac0

**Comment by Linkaiup** [comment]
could you please approve to run workflow for this PR? @lenboo @CalvinKirs

**Review by davidzollo — DISMISSED** [review]
+1
good job

**Comment by Linkaiup** [comment]
I fix the error Assert result to the right Assert result.Need to re-approve the PR? @dailidong

**Comment by zhongjiajie** [comment]
@CalvinKirs @lenboo @ruanwenjun @zhuangchong @JinyLeeChina Do you have time to approve the workflow run?

**Comment by kezhenxu94** [comment]
`AlertPluginManagerTest.java` will be removed via https://github.com/apache/dolphinscheduler/pull/6723 so I think this PR is unnecessary

**Comment by zhongjiajie** [comment]
> `AlertPluginManagerTest.java` will be removed via #6723 so I think this PR is unnecessary

That a bad news for our first time contributor 😭 , @Linkaiup maybe you could go and find other interested in green hand issue in https://github.com/apache/dolphinscheduler/issues/5689

**Comment by zhongjiajie** [comment]
@kezhenxu94 I found out after #6723  merged , test file still exists, did I miss something here? should this PR be closed or be merged?

**Comment by kezhenxu94** [comment]
> @kezhenxu94 I found out after #6723 merged , test file still exists, did I miss something here? should this PR be closed or be merged?

Where did you find the file `AlertPluginManagerTest.java`? It's gone in dev branch already.

Did you update the dev branch? @zhongjiajie

**Comment by zhongjiajie** [comment]
I go and see in https://github.com/apache/dolphinscheduler/pull/6674/files, the diff show the file still exists @kezhenxu94

**Comment by kezhenxu94** [comment]
> I go and see in https://github.com/apache/dolphinscheduler/pull/6674/files, the diff show the file still exists @kezhenxu94

The link you posted https://github.com/apache/dolphinscheduler/pull/6674/files is this PR itself.... the dev branch has removed the file

**Comment by zhongjiajie** [comment]
Before now, I thought that `diff` is show different between target branch and PR change. If the target file is deleted, the diff would show PR try to add all content to target branch

**Comment by Linkaiup** [comment]
> Before now, I thought that `diff` is show different between target branch and PR change. If the target file is deleted, the diff would show PR try to add all content to target branch

`diff` may be a snapshot of the moment I submitted the code?That a bad news.Thank you for your reply !

**Comment by zhongjiajie** [comment]
> > Before now, I thought that `diff` is show different between target branch and PR change. If the target file is deleted, the diff would show PR try to add all content to target branch
> 
> `diff` may be a snapshot of the moment I submitted the code?That a bad news.Thank you for your reply !

Yeah, you are right, It just a snapshot in PR diff. So I think maybe you should close this PR @Linkaiup , you could go and search other good first issue in  #5689

**Comment by Linkaiup** [comment]
> 
OK.I submitted a new pr #6940 .Looking forward to your code review!

**Comment by Linkaiup** [comment]
> > > Before now, I thought that `diff` is show different between target branch and PR change. If the target file is deleted, the diff would show PR try to add all content to target branch
> > 
> > 
> > `diff` may be a snapshot of the moment I submitted the code?That a bad news.Thank you for your reply !
> 
> Yeah, you are right, It just a snapshot in PR diff. So I think maybe you should close this PR @Linkaiup , you could go and search other good first issue in #5689

OK.I submitted a new pr #6940 .Looking forward to your code review!

**Pull request closed without merging**